tests/linalg_test.py

```diff
@@ -750,33 +751,40 @@ def testSolveTriangular(self, lower, transpose_a, unit_diagonal, lhs_shape,
 
   @parameterized.named_parameters(jtu.cases_from_list(
       {"testcase_name":
-       "_lhs={}_rhs={}_lower={}_transposea={}_unit_diagonal={}".format(
-           jtu.format_shape_dtype_string(lhs_shape, dtype),
-           jtu.format_shape_dtype_string(rhs_shape, dtype),
-           lower, transpose_a, unit_diagonal),
-       "lower": lower, "transpose_a": transpose_a,
-       "unit_diagonal": unit_diagonal, "lhs_shape": lhs_shape,
-       "rhs_shape": rhs_shape, "dtype": dtype, "rng": rng}
+       "_A={}_B={}_lower={}_transposea={}_conja={}_unitdiag={}_leftside={}".format(
+           jtu.format_shape_dtype_string(a_shape, dtype),
+           jtu.format_shape_dtype_string(b_shape, dtype),
+           lower, transpose_a, conjugate_a, unit_diagonal, left_side),
+       "lower": lower, "transpose_a": transpose_a, "conjugate_a": conjugate_a,
+       "unit_diagonal": unit_diagonal, "left_side": left_side,
+       "a_shape": a_shape, "b_shape": b_shape, "dtype": dtype, "rng": rng}
       for lower in [False, True]
       for unit_diagonal in [False, True]
       for dtype in float_types + complex_types
-      for transpose_a in (
-        [0, 1] if onp.issubdtype(dtype, np.floating) else [0, 1, 2])
-      for lhs_shape, rhs_shape in [
-          ((4, 4), (4,)),
```

Comment:
I note we no longer test the scipy wrapper at all, and the one useful thing it does is support vectors on the RHS.

Perhaps add another test just for that case?